deps/cmake/MatrixClient.cmake

```diff
@@ -8,6 +9,13 @@ if(APPLE)
     set(PLATFORM_FLAGS "-DOPENSSL_ROOT_DIR=/usr/local/opt/openssl")
 endif()
 
+# Force to build with the bundled version of Boost, if requested. This is
+# necessary because if an outdated Boost is installed, then CMake will grab
+# that instead of the bundled version of Boost, like we wanted.
+if(USE_BUNDLED_BOOST)
```

Comment:
You could remove this `if` statement because this file will only be included if that flag is `true` anyway.